### PR TITLE
Feature/filter location search

### LINKED ISSUE #91: Feed/Filter by Location - Zip Code Field
NOTE: This is ONLY for the zip code field portion of the UI - NOT the Location title, the - button, the Share my location, the Clear filters button, or the Apply filters button

ALSO NOTE: This does not include the functionality of actually searching for locations by zip code

* Form field with "Zip code, neighborhood or city"
* Tapping text field brings up a keyboard
* Localized

Screen Shot 2020-05-17 at 5 34 59 PM

### PR DESCRIPTION
**Pull request by ethanswift**
**What this PR does**:

Creates a location search cell for location filter modal

### Checklist

- [x] Compiler warnings resolved
- [x] Linter warnings resolved
- [x] User-facing strings in Localizable.strings file
- [x] Unused code, print statements, and comments removed

**Which issue(s) this PR fixes**:

Resolves #
when the user clicks the location button he/she can search the location via this text field 

**Special notes for reviewers**:

**Additional comments**:

### CODE REVIEW

File: FightPandemics/Features/Filters/LocationCellSearch.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+//  THE SOFTWARE.
+
+
+import UIKit
+
+class LocationCellSearch: UIView {
+    private var searchBar = UITextField()
+    func setUp() {
+        searchBar.attributedPlaceholder = NSAttributedString(string: "FilterLocationCellSearchBar".localized, attributes: [NSAttributedString.Key.font: Fonts.poppinsRegular.customFont(size: 16), NSAttributedString.Key.foregroundColor: UIColor(named: "#979797")!])
+        searchBar.backgroundColor = UIColor(named: "#F2F2F2")
```
Comment (stavares843) [inline FightPandemics/Features/Filters/LocationCellSearch.swift]: we have this color on the palette - fightPandemicsWhiteSmoke()

UIColor.fightPandemicsWhiteSmoke()

Comment (ethanswift) [reply FightPandemics/Features/Filters/LocationCellSearch.swift]: fixed

File: FightPandemics/Features/Filters/LocationCellSearch.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        magImg.frame = CGRect(x: 15, y: 15, width: 20, height: 20)
+        lView.frame = CGRect(x: 0, y: 0, width: 50, height: 50)
+        searchBar.leftViewMode = .always
+        searchBar.leftView = lView
+        searchBar.clipsToBounds = true
+        searchBar.layer.masksToBounds = true
+        searchBar.frame.size.height = 50
+        searchBar.layer.cornerRadius = searchBar.frame.size.height / 2
+        searchBar.makeSubview(of: self)
+            .width(UIScreen.main.bounds.width - 50)
```
Comment (OnyekachiSamuel) [inline FightPandemics/Features/Filters/LocationCellSearch.swift]: We don't actually need this width constraint, you have already constrained the view to left and right, that is enough already, the additional width constraint will introduce ambiguous constraint.

**Review by OnyekachiSamuel — CHANGES_REQUESTED** [review]

File: FightPandemics/Features/Filters/LocationCellSearch.swift (hunk 2 above)
Comment (ethanswift) [reply FightPandemics/Features/Filters/LocationCellSearch.swift]: fixed!

**Review by OnyekachiSamuel — APPROVED** [review]

**Pull request merged**